pkg/security/resolvers/sbom/file_querier.go

```diff
@@ -125,6 +125,12 @@ func (fq *fileQuerier) len() int {
 	return len(fq.files)
 }
 
+func (fq *fileQuerier) clear() {
+	fq.files = nil
```

Comment:
is it not going to break the cache mechanism ?

Reply:
Clear is only called on sboms whose `deleted` attribute is set to true, so they are not used for the cache